stackdriver/lib/stackdriver.rb

```diff
@@ -17,9 +17,9 @@
 gem "google-cloud-logging"
 gem "google-cloud-monitoring"
 
-require "google/cloud/error_reporting/v1beta1"
 require "google/cloud/logging"
-require "google/cloud/monitoring/v3"
+
+require "google/cloud/error_reporting/middleware" if defined? ::Rack
```

Comment:
It's almost certain that the caller will have already required "rack" in that case, but it's still prudent to do so here in this file as well.
I think this covers our cases now. Thanks!